guide_restli_server.md

```diff
@@ -875,29 +867,140 @@ Keys, query parameters, or action parameters.
 Keys:
 
 ```
-@RestLiCollection(name="entities",
-namespace = "com.example",
-keyTyperefClass = CustomObjectRef.class)
-public class EntitiesResource extends CollectionResourceTemplate<Urn,
-CustomObject>
+@RestLiCollection(name="entities", 
+                  namespace = "com.example",
+                  keyTyperefClass = CustomObjectRef.class)
+public class EntitiesResource extends CollectionResourceTemplate<CustomObject, Urn>
 ```
 
 Compound keys:
 
 ```
-RestLiAssociation(name="entities", namespace="com.example",
-                                   assocKeys={`Key(name="o",
-    type=CustomObject.class, typeref=CustomObjectRef.class)})
+@RestLiAssociation(name="entities", 
+                   namespace="com.example",
+                   assocKeys={@Key(name="o", type=CustomObject.class, typeref=CustomObjectRef.class)})
 ```
 
 Query parameters:
 
 ```
 @QueryParam(value="o", typeref=CustomObjectRef.class) CustomObject o
 
-@QueryParam(value="oArray", typeref=CustomObjectRef.class)
-CustomObject[] oArray
+@QueryParam(value="oArray", typeref=CustomObjectRef.class) CustomObject[] oArray
 ```
+<a id="BATCHFINDER"></a>
+
+#### BATCH FINDER
+The BATCH_FINDER allows combining multiple requests to one FINDER within a Rest.li resource into a single call. 
```

Comment:
This whole section seems a duplicate to the new batch finder document. The idea was to explain only the basic concept here and to provide all the details in the new document. In the end, we should provide a link to this new doc to guide the reader if she/he wants more details. Same comment in the "client" document.